conf/modules/board_matek_f765_car.xml

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="board_matek_f765_car" dir="boards">
```

Comment:
what is the difference with "wing" version ?

Reply:
There are no differences. I had to remove that file